plugins/jobs/plugin.zsh

```diff
@@ -0,0 +1,14 @@
+geometry_prompt_jobs_setup() {
+  return true
```

Comment:
splitting out the readme into user-focused, and contributing to dev-focused information seems good to me.